### PR TITLE
Use changed time for fetching new itinerary

### PR DESCRIPTION
**Pull request by hannesj**

### CODE REVIEW

File: app/component/navigation/index-sub-navigation.cjsx [hunk 1]
```diff
@@ -1,78 +1,22 @@
-React       = require 'react'
-TimeStore   = require '../../store/time-store' 
-TimeActions = require '../../action/time-action'
+React         = require 'react'
```
Comment (hannesj) [inline app/component/navigation/index-sub-navigation.cjsx]: These should probably be removed

Comment (tuukka) [inline app/component/navigation/index-sub-navigation.cjsx]: @hannesj What do you mean? Remove what from where?

Comment (hannesj) [inline app/component/navigation/index-sub-navigation.cjsx]: the requires on lines 2-4 are never used on the page

Comment (tuukka) [inline app/component/navigation/index-sub-navigation.cjsx]: I see, right. When I opened the link to the comment from chat, Github showed only line 1. "View full changes" helps :-)

**Pull request merged**